src/app/public/modules/data-manager/data-manager.service.ts

```diff
@@ -179,13 +184,25 @@ export class SkyDataManagerService implements OnDestroy {
    * @param sourceId The ID of the entity subscribing to data state updates. This can be any value you choose
    * but should be unique within the data manager instance and should also be used when that entity updates the state.
    */
-  public getDataStateUpdates(sourceId: string): Observable<SkyDataManagerState> {
+  public getDataStateUpdates(
+    sourceId: string,
+    updateFilter?: SkyDataManagerStateUpdateFilterArgs
+    ): Observable<SkyDataManagerState> {
```

Comment:
This should be outdented so the parenthesis is in the same column as the method declaration:

```
  public getDataStateUpdates(
    sourceId: string,
    updateFilter?: SkyDataManagerStateUpdateFilterArgs
  ): Observable<SkyDataManagerState> {
```